test_utils.go

```diff
@@ -25,24 +25,64 @@ func NewOverflowTest(t *testing.T) *OverflowTestUtils {
 	// return &OverflowTestUtils{T: t, O: overflow.NewOverflowEmulator().Start()}
 }
 
-func (otu *OverflowTestUtils) SetupTreasury(name string, signers []string) *OverflowTestUtils {
+func (otu *OverflowTestUtils) SetupTreasury(name string, signers []string, threshold uint64) *OverflowTestUtils {
 	addresses := make([]string, len(signers))
-	for _, name := range signers {
```

Comment:
This was buggy, created an array which is double the size because of appending instead of inserting on the right index, and then when the addresses were created based on that - 0x00000000 address was appearing and creating bugs for other tests - especially for adding/removing signers

Reply:
I was wondering why i was seeing the 0x000000 address!! good catch